data-imports/utils.py

```diff
@@ -152,7 +152,7 @@ def hpd_csv2sql(description, input_csv_url, sep_char,
             log.info("Processing chunk {}".format(chunk_num))
 
             chunk_pickle_file = pickle_file.format(chunk_num)
-            df = process_df(df, log, description, save_pickle, chunk_pickle_file,
+            df = process_df(df, log, description, save_pickle, pickle_file,
```

Comment:
Hmm, I wonder if there was a bad merge somewhere? Or if that param order is in a different PR? The changes I made here seem to correspond to what's in master now for `process_df`.

https://github.com/chrishenry/deltanyc-heatseek/blob/master/data-imports/utils.py#L192